docs/other-topics/naming-strategies.md

```diff
@@ -4,13 +4,13 @@ title: Naming Strategies
 
 ## The `underscored` option
 
-Sequelize provides the `underscored` option for a model. When `true`, this option will set the `field` option on all attributes to the [snake_case](https://en.wikipedia.org/wiki/Snake_case) version of its name. This also applies to foreign keys automatically generated by associations and other automatically generated fields. Example:
+Sequelize provides the `underscored` option for a model. When `true`, this option will set the `field` option on all attributes to the [snake_case](https://en.wikipedia.org/wiki/Snake_case) version of its name. This also applies to table names and foreign keys automatically generated by associations and other automatically generated fields. Example:

```

Comment:
I think it's important to mention that this only applies if the `tableName` has not been provided. If it was provided, it's used as-is

Reply:
```suggestion
Sequelize provides the `underscored` option for a model. When `true`, this option will set the `field` option on all attributes to the [snake_case](https://en.wikipedia.org/wiki/Snake_case) version of its name (unless manually set). This also applies to table names and foreign keys automatically generated by associations and other automatically generated fields. Example:

```

Reply:
Yes that sounds good to me :)

Reply:
yeah, cause it actually applies to fields as well, right? you can manually set those too.

Reply:
@barraponto can you still commit this change?

Reply:
I was under the impression it was commited already 🤯 

done.